definitions/ext-host/golden_metrics.yml

```diff
@@ -1,9 +1,47 @@
 systemLoad:
   title: System Load
-  query:
-    select: sum(datadog.system.load.15)
+  queries:
+    datadog:
+      select: sum(datadog.system.load.15)
+    flexagent:
+      select: latest(loadAverageFifteenMinute)
+      from: FlexSystemSample
+      eventId: entity.guid
+
+cpuPercent:
+  title: CPU Percent
+  unit: PERCENTAGE
+  queries:
+    flexagent:
```

Comment:
we must provide a query for each provider.

Reply:
A host is a host in itself. The golden metrics of a host are always the same regardless of how they are queried internally.

In this case you have provided the queries for the existing golden metrics just fine.
Since you added a new one we need to also add the query for the existing sources, in this case datadog.

As far as I could find this is the most likely candidate `max(datadog.system.cpu.system)`.
And if someone complains we are happy to change the query so please add it for datadog.

Reply:
added the dd queries